### ISSUE TITLE
Login Screen 

### ISSUE THREAD

**Issue by HimeshNayak** [HimeshNayak/SwingerIOT#1]
Here are some of the issues generated from Axe-Core DevTools.
```
[
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(4) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 2.59 (foreground color: #ffffff, background color: #64b174, font size: 15.0pt (20px), font weight: bold). Expected contrast ratio of 3:1",
        "source": "<p style=\"text-align: center; line-height: 1.17188; font-size: 20px; font-weight: bold; font-family: Poppins, Arial, sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 23.2px; width: 281px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 67, 656); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:27:29.894Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697#/GeneratedLoginWidget",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(6) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 2.59 (foreground color: #ffffff, background color: #64b174, font size: 15.0pt (20px), font weight: bold). Expected contrast ratio of 3:1",
        "source": "<p style=\"text-align: center; line-height: 1.17188; font-size: 20px; font-weight: bold; font-family: Poppins, Arial, sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 23.2px; width: 281px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 67, 560); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:27:29.894Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697#/GeneratedLoginWidget",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(7) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 1.48 (foreground color: #ffffff, background color: #9be5aa, font size: 26.3pt (35px), font weight: bold). Expected contrast ratio of 3:1",
        "source": "<p style=\"text-align: left; line-height: 1.17188; font-size: 35px; font-weight: bold; font-family: Poppins, Arial, sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 40.8px; width: 258px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 80, 66); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:27:29.894Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697#/GeneratedLoginWidget",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "needsReview": true,
        "isManual": false,
        "selector": [
            "flt-picture > p"
        ],
        "summary": "Fix any of the following:\n  Element's background color could not be determined because it's partially obscured by another element",
        "source": "<p style=\"text-align: center; line-height: 1; font-size: 10px; font-weight: 900; font-family: sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 9.6px; width: 80px; transform-origin: 0px 0px 0px; transform: matrix(0.707107, 0.707107, -0.707107, 0.707107, 364.81, -7.77817);\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:27:29.894Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697#/GeneratedLoginWidget",
        "foundBy": "himeshnayak015@gmail.com"
    }
]
```
